tests/ui/suspicious_arguments.stderr

```diff
@@ -0,0 +1,102 @@
+  --> $DIR/suspicious_arguments.rs:44:12
+   |
+LL |     resize(height(), width());
+   |            ^^^^^^    ^^^^^
+   |
+   = note: `-D clippy::suspicious-arguments` implied by `-D warnings`
+note: the arguments are defined here
+  --> $DIR/suspicious_arguments.rs:17:11
+   |
+LL | fn resize(width: usize, height: usize) {}
+   |           ^^^^^         ^^^^^^
+
+error: these arguments are possibly swapped
+  --> $DIR/suspicious_arguments.rs:53:17
+   |
+LL |     resize(uwu::height(), uwu::width());
+   |                 ^^^^^^         ^^^^^
+   |
+note: the arguments are defined here
+  --> $DIR/suspicious_arguments.rs:17:11
+   |
+LL | fn resize(width: usize, height: usize) {}
+   |           ^^^^^         ^^^^^^
+
+error: these arguments are possibly swapped
+  --> $DIR/suspicious_arguments.rs:60:12
+   |
+LL |     resize(height, width);
+   |            ^^^^^^  ^^^^^
+   |
+note: the arguments are defined here
+  --> $DIR/suspicious_arguments.rs:17:11
+   |
+LL | fn resize(width: usize, height: usize) {}
+   |           ^^^^^         ^^^^^^
+
+error: these arguments are possibly swapped
+  --> $DIR/suspicious_arguments.rs:61:17
+   |
+LL |     Bitmap::new(height, width);
+   |                 ^^^^^^  ^^^^^
+   |
+note: the arguments are defined here
+  --> $DIR/suspicious_arguments.rs:22:12
+   |
+LL |     fn new(width: usize, height: usize) -> Self {
+   |            ^^^^^         ^^^^^^
+
+error: these arguments are possibly swapped
+  --> $DIR/suspicious_arguments.rs:67:17
+   |
+LL |     resize(dims.height, dims.width);
+   |                 ^^^^^^       ^^^^^
+   |
+note: the arguments are defined here
+  --> $DIR/suspicious_arguments.rs:17:11
+   |
+LL | fn resize(width: usize, height: usize) {}
+   |           ^^^^^         ^^^^^^
+
+error: these arguments are possibly swapped
+  --> $DIR/suspicious_arguments.rs:71:10
+   |
+LL |         .height,
+   |          ^^^^^^
+...
+LL |         .width);
+   |          ^^^^^
+   |
+note: the arguments are defined here
+  --> $DIR/suspicious_arguments.rs:17:11
+   |
+LL | fn resize(width: usize, height: usize) {}
+   |           ^^^^^         ^^^^^^
+
+error: these arguments are possibly swapped
+  --> $DIR/suspicious_arguments.rs:93:20
+   |
+LL |     itertools::all(f, iterable);
+   |                    ^  ^^^^^^^^
+   |
+note: the arguments are defined here
+  --> /home/jess/.cargo/registry/src/github.com-1ecc6299db9ec823/itertools-0.10.1/src/free.rs:151:18
+   |
+LL | pub fn all<I, F>(iterable: I, f: F) -> bool
+   |                  ^^^^^^^^     ^
+
+error: these arguments are possibly swapped
+  --> $DIR/suspicious_arguments.rs:102:20
+   |
+LL |     std::mem::swap(y, x);
+   |                    ^  ^
+   |
+note: the arguments are defined here
+  --> /home/jess/.rustup/toolchains/nightly-2022-07-28-x86_64-unknown-linux-gnu/lib/rustlib/src/rust/library/core/src/mem/mod.rs:711:22
+   |
+LL | pub const fn swap<T>(x: &mut T, y: &mut T) {
+   |                      ^          ^
```

Comment:
You can use [an `// aux-build` comment](https://rustc-dev-guide.rust-lang.org/tests/compiletest.html?highlight=aux-build#building-auxiliary-crates) to test the separate crate version without running into these path issues